docker/candlepin-base-cs9/setup-devel-env.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+#
+# Sets a system up for a candlepin development environment (minus a db,
+# handled separately), and an initial clone of candlepin.
+
+set -ve
+
+source /root/dockerlib.sh
+
+export JAVA_VERSION=1.8.0
```

Comment:
Set this to 11. We don't have any supported branches that require 1.8 any more.

Reply:
done